sdk/nodejs/automation/stack.ts

```diff
@@ -761,6 +761,12 @@ function applyGlobalOpts(opts: GlobalOpts, args: string[]) {
     if (opts.debug) {
         args.push("--debug");
     }
+    if (opts.suppressOutputs) {
+        args.push("--suppress-outputs");
```

Comment:
I was wondering if this is correct, since `suppress-progress` and `suppress-outputs` are only accepted for some pulumi commands.  However it looks like we're only runnining `applyGlobalOpts` for `up`, `preview`, `refresh` and `destroy`, which all do support these flags, so this seems okay.